### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by abhinav**
Forgot to add entry for #225 and update release date when 0.2.0 was released.

CC @kriskowal @breerly @bombela

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 Releases
 ========
 
-v0.2.0 (unreleased)
+v0.3.0 (unreleased)
+------------------
+
+-   **Breaking**: Most keywords reserved by Apache Thrift are now disallowed as
```
Comment (breerly) [inline CHANGELOG.md]: What do you "most"?

Comment (abhinav) [inline CHANGELOG.md]: It's really "all" but if they add more in the future, this is no longer true. But you're right. I'll change this.

**Comment by breerly** [comment]
Why not export Version?

**Comment by abhinav** [comment]
I'll export it eventually in a submodule. Nobody should be importing the `main` module directly.

**Pull request merged**